jetbrains-core/src/software/aws/toolkits/jetbrains/core/gettingstarted/OuterPanel.kt

```diff
@@ -0,0 +1,423 @@
+
+    init {
+        addToCenter(
+            panel {
+                indent {
+
+                    row {
+                        label(panelTitle).applyToComponent { font = JBFont.h3() }
+                    }
+
+                    row {
+                        text(Paneltext.COMMIT_ICON)
+                        panel {
+                            row(message("iam_identity_center.name")) {
+                            }.rowComment(message("aws.onboarding.getstarted.panel.idc_row_comment_text"))
+                        }
+                    }
+
+                    row {
+                        text(Paneltext.CANCEL_ICON)
+                        panel {
+                            row(message("aws_builder_id.service_name")) {
+                            }.rowComment(message("aws.getstarted.auth.panel.notSupport_text"))
+                        }
+                    }
+
+                    row {
+                        text(Paneltext.COMMIT_ICON)
+                        panel {
+                            row(message("settings.credentials.iam")) {
+                            }.rowComment(message("aws.getstarted.auth.panel.notSupport_text"))
+                        }
+                    }
+                }
+            }
+        )
+
+        border = IdeBorderFactory.createRoundedBorder().apply {
+            setColor(CodeWhispererColorUtil.POPUP_BUTTON_BORDER)
+            preferredSize = Dimension(Paneltext.PANEL_WIDTH, Paneltext.BULLET_PANEL_HEIGHT)
+        }
+    }
+}
+
+class CodeCatalystAuthBullets(private var panelTitle: String) : BorderLayoutPanel() {
+
+    init {
+        addToCenter(
+            panel {
+                indent {
+
+                    row {
+                        label(panelTitle).applyToComponent { font = JBFont.h3() }
+                    }
+
+                    row {
+                        text(Paneltext.CANCEL_ICON)
+                        panel {
+                            row(message("iam_identity_center.name")) {
+                            }.rowComment(message("aws.getstarted.auth.panel.notSupport_text"))
+                        }
+                    }
+
+                    row {
+                        text(Paneltext.COMMIT_ICON)
+                        panel {
+                            row(message("aws_builder_id.service_name")) {
+                            }.rowComment("Need to insert tagline")
+                        }
+                    }
+
+                    row {
+                        text(Paneltext.CANCEL_ICON)
+                        panel {
+                            row(message("settings.credentials.iam")) {
+                            }.rowComment(message("aws.getstarted.auth.panel.notSupport_text"))
+                        }
+                    }
+                }
+            }
+        )
+
+        border = IdeBorderFactory.createRoundedBorder().apply {
+            setColor(CodeWhispererColorUtil.POPUP_BUTTON_BORDER)
```

Comment:
does this work
```suggestion
            color = CodeWhispererColorUtil.POPUP_BUTTON_BORDER
```
? also why are we using the codewhisperer color utils?

Reply:
done